mrbgems/mruby-bin-utils/README.md

```diff
@@ -0,0 +1,29 @@
+# mruby-bin-utils
+
+This is a command line program that serves as a wrapper to compile and link source code requiring libmruby.
+
+The name is a play on GNU Binutils.
+
+## How to build and use it
+
+Add this mruby-bin-utils to your build configuration file and build mruby.
+
+```ruby
+# in your build configuration file
+
+MRuby::Build.new do |conf|
+  ...
+  conf.gem core: "mruby-bin-utils"
+  ...
+end
+```
+
+Then compile and link your code in the generated command line program.
```

Comment:
I think this line needs to be updated. You need to describe clearly what we get from these command-lines.

Reply:
Modified.